src/midisequencer.cpp

```diff
@@ -73,8 +73,15 @@ namespace midisequencer{
     float sequencer::rewind_to_loop()
     {
         position = loop_position;
+        if (position != messages.begin()) {
+            position--;
```

Comment:
> Via Twitter:
> "EasyRPG's MIDI playback doesn't read the last 1 tick note off just before the loop."

Was that twitter user running this branch? I remember experiencing this back when I was working on #2245 but I was pretty sure I fixed it in this branch already with this line

Reply:
No this was reported by someone using the web player on atsumaru. So is a bug in master. Was just not sure if this PR fixes this